ecodse_funtime_alpha/train.py

```diff
@@ -16,24 +19,62 @@ def train_loop(dataset, model, optimizer):
         print(loss)
 
 
-def fit_loop(dataset, model, optimizer):
-    dataset = dataset.shuffle(12)
-    dataset = dataset.repeat(20)
-    dataset = dataset.batch(4)
+def fit_loop(dataset, model, optimizer, nepoch, batchsize):
+    nstep = len(list(dataset)) // batchsize
+    dataset = dataset.shuffle(12)  # 12 = buffer size
```

Comment:
Advice # 42. When the code can be made clearer don't add a comment for clarification.

dataset = dataset.shuffle(buffer_size=12)

Reply:
solved.